gazebo_plugins/src/gazebo_ros_imu_sensor.cpp

```diff
@@ -69,6 +69,24 @@ void GazeboRosImuSensor::Load(gazebo::sensors::SensorPtr _sensor, sdf::ElementPt
     return;
   }
 
+  bool initial_orientation_as_reference = false;
+  if (!_sdf->HasElement("initial_orientation_as_reference")) {
+    RCLCPP_INFO_STREAM(impl_->ros_node_->get_logger(),
```

Comment:
The uncrustify config changed; now we should break after the open paren.
```suggestion
    RCLCPP_INFO_STREAM(
      impl_->ros_node_->get_logger(),
```

See https://ci.ros2.org/job/ci_linux/10952/testReport/junit/gazebo_plugins/uncrustify/src_gazebo_ros_imu_sensor_cpp/

Reply:
02aa68b